libp2p/Common.h

```diff
@@ -116,7 +116,7 @@ typedef std::pair<std::string, u256> CapDesc;
 typedef std::set<CapDesc> CapDescSet;
 typedef std::vector<CapDesc> CapDescs;
 
-struct PeerInfo
+struct PeerSessionInfo
```

Comment:
Afaik, this code won't be reviewed by auditors. This struct contains session info about the peer, as is described by the properties of the struct. Notes aren't touched by this PR, so I'm not sure. Docs, of course, are always helpful.